### PR TITLE
Change riot.im link to matrix.to

### PR DESCRIPTION
**Pull request by resynth1943**

### CODE REVIEW

File: templates/base.rs.html [hunk 1]
```diff
@@ -93,7 +93,7 @@ <h3>@Instance::get_local().map(|i| i.name).unwrap_or_default()</h3>
                 <h3>@concat!("Plume ", env!("CARGO_PKG_VERSION"))</h3>
                 <a href="https://docs.joinplu.me">@i18n!(ctx.1, "Documentation")</a>
                 <a href="https://github.com/Plume-org/Plume">@i18n!(ctx.1, "Source code")</a>
-                <a href="https://riot.im/app/#/room/#plume:disroot.org">@i18n!(ctx.1, "Matrix room")</a>
+                <a href="https://matrix.to/#/#plume:disroot.org">@i18n!(ctx.1, "Matrix room")</a>
```
Comment (igalic) [inline templates/base.rs.html:96]: we should also move from distroot.io to… something that exists…

**Review by igalic — COMMENTED** [review]
🙋🏻‍♀️

File: templates/base.rs.html (hunk 1 above)
Comment (elegaanz) [reply templates/base.rs.html:96]: This is just a room alias, matrix rooms are not really linked to a homeserver I think, so there is no need to create a new room, just to add a new alias (and I believe there already are some other aliases for the Plume rooms).

Comment (resynth1943) [reply templates/base.rs.html:96]: Send me another address, and I'll add it in.

Comment (KitaitiMakoto) [reply templates/base.rs.html:96]: You may use `#plume-blog:matrix.org` as an alias.

Comment (resynth1943) [reply templates/base.rs.html]: See 07a6be4.

**Review by igalic — APPROVED** [review]

**Comment by KitaitiMakoto** [comment]
Thank you!

**Pull request merged**